src/model/affiliation.ts

```diff
@@ -47,6 +48,33 @@ export class Affiliation extends BackmatterEntity {
     this.institution = json.institution as { name: string } || {name: ''};
     this.address = json.address as { city: string } || {city: ''};
   }
+
+  public toXml(): Element {
+    const xmlDoc = new DOMImplementation().createDocument(null, null); 
+    const affEl = xmlDoc.createElement('aff');
+    affEl.setAttribute('id', this.id);
+  
+    const label = xmlDoc.createElement('label');
+    label.appendChild(xmlDoc.createTextNode(this.label || ''));
+    affEl.appendChild(label);
+  
+    const department = xmlDoc.createElement('institution');
+    department.appendChild(xmlDoc.createTextNode(get(this, 'institution.name', '')));
+    affEl.appendChild(department);
+  
+    const addressLine = xmlDoc.createElement('addr-line');
+    const city = xmlDoc.createElement('named-content');
+    city.setAttribute('content-type', 'city');
+    city.appendChild(xmlDoc.createTextNode(this.address?.city || ''));
+    addressLine.appendChild(city);
+    affEl.appendChild(addressLine);
+  
+    const country = xmlDoc.createElement('country');
```

Comment:
Just cross checked this against the spec, seems that there should be an attribute `country` here.

Reply:
@FAtherden-eLife 

> The value of the country attribute should be one of the ISO_3166-1 2 letter country codes, although the text inside the country element does not need to follow the respective English name in that list.

We currently have a free text input for affiliation country so this isn't going to be something I can 100% accurately detect from the string. This gives us a couple of options 

- We replace the input with a dropdown select containing all countries in the ISO_3166-1 2 letter country code list.
- We add another input to the affiliation modal for selecting the correct ISO_3166-1 2 letter country code
- We do a look up with what we have and if we find a match assign and if not just go without
- We never assign ISO_3166-1 2 letter country code attribute for consistency

Either way I'd prefer to raise a new issue for this. As everything other than the last option will require us creating a lookup table which goes beyond the scope of this refactor.

Reply:
@Cooryd, yup, absolutely fine. IMO we should go with both of these options (when we tackle that work):
> - We add another input to the affiliation modal for selecting the correct ISO_3166-1 2 letter country code
> - We do a look up with what we have and if we find a match assign and if not just go without

Reply:
I'll leave as is for now then and create a new issue to add ISO_3166-1 2 letter country code support to editor. We'll need to discuss the specifics of how this should work as part of this issue.

Reply:
https://github.com/libero/editor/issues/719